src/components/LoadingBox/LoadingBox.style.less

```diff
@@ -0,0 +1,51 @@
+.skeleton-task-box {
+  // position: absolute;
```

Comment:
@SchafferCPqD está comentado para usar futuramente? se for, deixe um comentário explicando, por favor